libs/elements/src/side-navigation/vl-side-navigation.element.cy.ts

```diff
@@ -0,0 +1,439 @@
+                                                        <a href="#content-2-1" data-vl-parent="content-2"
+                                                            >content 2 - 1</a
+                                                        >
+                                                    </div>
+                                                </li>
+                                                <li is="vl-side-navigation-item">
+                                                    <div>
+                                                        <a href="#content-2-2" data-vl-parent="content-2"
+                                                            >content 2 - 2</a
+                                                        >
+                                                    </div>
+                                                </li>
+                                                <li is="vl-side-navigation-item">
+                                                    <div>
+                                                        <a href="#content-2-3" data-vl-parent="content-2"
+                                                            >content 2 - 3</a
+                                                        >
+                                                    </div>
+                                                </li>
+                                                <li is="vl-side-navigation-item">
+                                                    <div>
+                                                        <a href="#content-2-4" data-vl-parent="content-2"
+                                                            >content 2 - 4</a
+                                                        >
+                                                    </div>
+                                                </li>
+                                            </ul>
+                                        </li>
+                                        <li is="vl-side-navigation-item" data-vl-parent="content-3">
+                                            <a
+                                                is="vl-side-navigation-toggle"
+                                                href="#content-3"
+                                                data-vl-child="content-3"
+                                            >
+                                                content 3
+                                                <i class="vl-vi vl-vi-arrow-right-fat"></i>
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </div>
+                            </nav>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        `);
+    });
+
+    it('should mount', () => {
+        cy.get('nav[is="vl-side-navigation"]');
```

Comment:
Beetje jammer dat we hier niet op de custom tag kunnen zoeken; (stel dat men onderliggend de html zou wijzigen gaat dit breken)

maar is intrensiek door lit een probleem =( dus ik snap het

Reply:
ja op een bepaald moment converteren we alles naar Lit, dan kunnen we idd query doen op tag zelf ipv `is="..."`

Reply:
Dat komt inderdaad doordat het een element is en geen component, ooit wordt dit omgevormd zodat we enkel nog maar web components hebben.